### ISSUE TITLE
Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by jpdevries** [Sterc/mediamanager#31]
The MODX Default theme does not meet WCAG Success Criteria, so I do not think it is crucial for the Media Manager to either until MODX does (4.x). That said, I'll log any accessibility issues here along with proposed solutions and try to submit PRs for them as well.

![](http://j4p.us/0E2Y0d3Y082H/Screen%20Shot%202017-07-05%20at%2011.59.09%20AM.png)

### Solution

Darken the green background until it meets sufficient color contrast (currently 2.48 needs to be 4.0).

**Comment by jpdevries** [comment]
These elements also don't meet WCAG contrast

![](http://j4p.us/3E3j3Y322321/Screen%20Shot%202017-07-05%20at%2012.12.54%20PM.png)

![](http://j4p.us/2S3g2B1h0m3C/Screen%20Shot%202017-07-05%20at%2012.13.42%20PM.png)
